VMS.Domain/Models/Favorite.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace VMS.Domain.Models
+{
+    public class Favorite
```

Comment:
Please make this class inherit from BaseEntity, see other models for references.

Reply:
done